backend/buildeR.DAL/Context/ModelBuilderExtensions.cs

```diff
@@ -8,8 +8,15 @@ public static class ModelBuilderExtensions
     {
         public static void Configure(this ModelBuilder modelBuilder)
         {
-            // apply configurations from EntityConfigurations folder
-           // modelBuilder.ApplyConfigurationsFromAssembly(typeof(UserConfig).Assembly);
+            modelBuilder.ApplyConfigurationsFromAssembly(typeof(BuildHistoryConfig).Assembly);
+            modelBuilder.ApplyConfigurationsFromAssembly(typeof(BuildPluginParameterConfig).Assembly);
+            modelBuilder.ApplyConfigurationsFromAssembly(typeof(BuildStepConfig).Assembly);
+            modelBuilder.ApplyConfigurationsFromAssembly(typeof(PluginCommandConfig).Assembly);
+            modelBuilder.ApplyConfigurationsFromAssembly(typeof(ProjectConfig).Assembly);
+            modelBuilder.ApplyConfigurationsFromAssembly(typeof(ProjectGroupConfig).Assembly);
+            modelBuilder.ApplyConfigurationsFromAssembly(typeof(ProjectTriggerConfig).Assembly);
+            modelBuilder.ApplyConfigurationsFromAssembly(typeof(TeamMemberConfig).Assembly);
+            modelBuilder.ApplyConfigurationsFromAssembly(typeof(UserSocialNetworkConfig).Assembly);
```

Comment:
This is excessive. `ApplyConfigurationsFromAssembly` applies configuration from all config files in the assembly.

Call only one of them

Reply:
So If I rewrite it to
modelBuilder.ApplyConfigurationsFromAssembly(typeof(BuildHistoryConfig).Assembly);
it will take configs from other assemblies?

Reply:
`typeof(BuildHistoryConfig).Assembly` result in **buildeR.DAL** assembly. Then, `ApplyConfigurationsFromAssembly` searches for classes in it that extends from IEntityTypeConfiguration<T> and calls its `Configure` method. As a result, it applies all configurations from EntityConfiguration folder

Reply:
Ok, so I'll leave only this line